activepieces/activepieces
# Delegating Property Values

## What does this PR do?

Allows people to assign dropdown, object, array and JSON piece properties to dynamic values.
Fixes # (issue)
1) Addresses #360 
2) Disables piece trigger forms and full screen editor in read only mode.

## Type of change




## How should this be tested?
Add a piece action -> toggle an input to a dynamic value -> input your value -> test flow and make sure the value passed was correct.
